woocommerce-google-analytics-integration.php

```diff
@@ -134,7 +134,12 @@ public function show_ga_pro_notices() {
 			$completed_orders = wc_orders_count( 'completed' );
 
 			// Only show the notice if there are 10 <= completed orders <= 100.
-			if ( ! ( 10 <= $completed_orders && $completed_orders <= 100 ) ) {
+			$too_few_orders_to_show_the_notice = $completed_orders < 10;
+			$too_many_orders_to_show_the_notice = $completed_orders > 100;
```

Comment:
```suggestion
			$too_few_orders_to_show_the_notice  = $completed_orders < 10;
			$too_many_orders_to_show_the_notice = $completed_orders > 100;
```
Align variable assignments.

Reply:
Fixed in c9449d8